models/double_entry_transactions/int_quickbooks__credit_card_pymt_double_entry.sql

```diff
@@ -0,0 +1,56 @@
+/*
+Table that creates a debit record to the associated bank account and a credit record to the specified credit card account.
```

Comment:
could you explain why we need double records? just for my own edification lol

Reply:
Sure thing! Essentially we need to create two records to account for the debit and credit offsets of this transaction. The transaction doesn't just have an impact once. Rather it will add in one place and remove from another. Typically this is shown when increasing an asset (buying new computers) while also increasing a liability (you paid with a credit card so incurred more credit).

This double entry method of accounting is called the matching principle and ensure you match a transaction to the impacted accounts within your business.